src/component/Flow.tsx

```diff
@@ -330,9 +340,30 @@ export default class Flow extends React.Component<FlowProps, FlowState> {
 
             ghost.actions.push(replyAction);
         } else {
+            // coun't up all our msg waits
+            let responseNumber = 1;
```

Comment:
should these be broken-out into helper methods, e.g. `getResponseNumber` and `getResultName`? would that make testing easier?